docs/_posts/2018-04-23-moving-to-modern-cpp.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: blogpost
+title:  "Open GEE moves to modern C++"
+section_id: blog
+date:   2018-04-23 13:30:00
+author: Open GEE Development Team
+---
+
+Hello Everyone!
+
+The Open GEE Development Team would like to make a special announcement about an
+effort to upgrade our usage of `C++` to a more modern standard. As of release
+`5.2.3`, we will begin compiling Open GEE code using the `C++11` standard. We are
```

Comment:
Before "We are ...".. may be add a phrase or two explaining why we want to move to the new compiler...:
"C++11 is the first major update of the C++ standard since 1998. C++11 features aim to promote writing clean, compact, type-safe and fast code".